tests/playwright-tests/utils/env.ts

```diff
@@ -0,0 +1,7 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+// eslint-disable-next-line import/prefer-default-export
+export class environment {
+  public static BASE_URL = process.env.PUBLICURL;
```

Comment:
Tiny thing, but generally we use underscores between words in these files, so that `PUBLICURL` becomes `PUBLIC_URL`, much in the same way that you have `BASE_URL` already.

Reply:
Also, I suspect it'll currently fail in the pipeline because it doesn't have a .env file from which to construct its BASE_URL, so maybe have a default fallback here like:

```
public static BASE_URL = process.env.PUBLIC_URL ?? "https://dev.explore-education-statistics.service.gov.uk/";
```

Reply:
Although this'll probably fail right now too because of basic auth.  We'd probably need to set up some pipeline variables which aren't checked in that you could then combine with the URL above to get past Dev's basic auth.